streams/src/test/java/org/apache/kafka/streams/processor/internals/ProcessorTopologyTest.java

```diff
@@ -1568,21 +1105,19 @@ private Topology createSimpleTopologyWithDroppingPartitioner() {
                 .addSink("sink", OUTPUT_TOPIC_1, new DroppingPartitioner(), "processor");
     }
 
-    @Deprecated // testing old PAPI
```

Comment:
This annotation was wrong -- should have been `@SuppressWarning("deprecation")`

Updating this code to use new Processor API now.